src/niveristand/_translation/utils.py

```diff
@@ -92,8 +115,24 @@ def get_variable_name_from_node(node):
 
 
 def get_channel_name(node):
-    if type(node) is ast.Str:
+    if sys.version_info < (3, 8) and type(node) is ast.Str:
         channel_name = node.s
+    elif check_ast_constant_str(node):
+        channel_name = node.value
     else:
         raise errors.TranslateError(_errormessages.invalid_type_for_channel_ref)
     return channel_name
+
+
+def check_ast_constant_str(node):
```

Comment:
Please modify these check_ast_constant_* methods to do the version checking as well and use them above to check if a node is of that type.E.g.

```
def is_node_ast_str(node):
    if sys.version_info < (3, 8) and type(node) is ast.Str or isinstance(node, ast.Constant) and isinstance(node.value, str):
        return true
    else:
        return false
```

or something similar. Also please create methods to extract the value from the node based on its type. E.g.

```
def get_value_from_str_node(node):
    if sys.version_info < (3, 8):
        return node.s
    else:
        return node.value
```
or something similar. If it makes more sense you can combine them, but I think having clear methods to extract the value from different AST nodes will make it easier to understand.